domain-services/ifix-master-data-service/src/test/java/org/egov/producer/ProducerTest.java

```diff
@@ -0,0 +1,43 @@
+package org.egov.producer;
```

Comment:
Are we using kafka in the master data service? 
If not delete everything related to kafka from the project.